reg_tests/chgres_cube/25km.conus.gfs.grib2.sh

```diff
@@ -24,7 +24,7 @@ export VARMAP_FILE=${HOMEufs}/parm/varmap_tables/GSDphys_noaero_var_map.txt
 export INPUT_TYPE='grib2'
 export CONVERT_NST=".false."
 export THOMPSON_AEROSOL_FILE=${HOMEufs}/fix/am/Thompson_MP_MONTHLY_CLIMO.nc
-export OROG_FILES_TARGET_GRID="C403_oro_data.tile7.halo4.nc"
+export OROG_FILES_TARGET_GRID="C405_oro_data.tile7.nc"
 export REGIONAL=1
```

Comment:
This version of chgres requires that all surface climo fields (like veg type) be defined at all points with at least some land (instead of 50% land). So, I had to regenerate all the fixed data for the regression tests. Using what I thought are the grid specs for the regional grids, I was unable to get the CRES value to match. After a few attempts, I ended up using the closest one. The original files were generated three years ago, so code and library changes are probably a factor. But the new files are very close to the original ones. If you visualize both, they look like the identical grids.

Reply:
Thanks!